### ISSUE TITLE
Could you provide the logfile of pretraining process?

### ISSUE THREAD

**Issue by forjiuzhou** [ChenRocks/UNITER#8]
Hi, 
I want to get an understanding of what happened during the pretraining, especially the train and validation loss change.
Thank you.
